app/controllers/mixins/breadcrumbs_mixin.rb

```diff
@@ -33,33 +34,58 @@ def data_for_breadcrumbs
           breadcrumbs.push(:title => @title)
         end
       else
+        self.x_node_text = params[:text] if action_name == "tree_select" # get text of the active node
+
         # Append breadcrumb from title of the accordion (eg "Policies")
-        breadcrumbs.push(:title => accord_title, :key => "#{accord_name}_accord", :action => "accordion_select") if accord_title
+        breadcrumbs.push(:title => accord_title, :key => "root") if accord_title
+
+        # On special page (tagitems, politems, etc.)
+        if @tagitems || @politems || @ownershipitems || @retireitems
+          breadcrumbs.push(special_page_breadcrumb(@tagitems || @politems || @ownershipitems || @retireitems, true)) unless options[:hide_special_item]
```

Comment:
I guess the first argument here is always `true` based on the `if` above. So no need to `||` twice.

Reply:
The variable which is present is also used in the method.